.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -45,7 +46,6 @@ body:
         - 'SWC transpilation'
         - 'Turbopack (--turbo)'
         - 'TypeScript'
-        - 'Other'
```

Comment:
The dropdown is optional, besides there is a default "None" added by GitHub, so this is not necessary.

Reply:
Weird. I wouldn't expect that GitHub adds none when its `multiple: true`.